.changeset/happy-peaches-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed issue with livechat agents not being able to leave omnichannel rooms if joining after a room has been closed (due to race conditions)
```

Comment:
You probably have to put here that if you get stuck the first time calling, then calling a 2nd time should fix the data

Reply:
I believe that's not exacty the case for this fix. From the reprorts we got, this only happens when the room has been closed by the visitor, not when the agent itself had already called it once before
I'll make this clear in the changeset